imagetest/test_suites/hostkey/hostkey_pubish_test.go

```diff
@@ -0,0 +1,60 @@
+// +build cit
+
+package hostkey
+
+import (
+	"flag"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+var (
+	runtest = flag.Bool("runtest", false, "really run the test")
```

Comment:
Where does this get used? Also, can you update the description to describe the test.

Reply:
this was from our legacy protection mechanism and should just be removed